hooks/pre/30-upgrade.rb

```diff
@@ -40,6 +40,14 @@ def migrate_foreman
   Kafo::Helpers.execute('foreman-rake db:migrate')
 end
 
+def remove_nodes_importers
+  Kafo::Helpers.execute("mongo pulp_database --eval 'db.repo_importers.remove({'importer_type_id': \"nodes_http_importer\"});'")
+end
+
+def remove_nodes_distributors
+  Kafo::Helpers.execute("mongo pulp_database --eval  'db.repo_distributors.remove({'distributor_type_id': \"nodes_http_distributor\"});'")
```

Comment:
There is no way to do this through a Rake task using Runcible to update repositories within the Katello repository?